common/ethnicities/wc_ethnicities_orcish.txt

```diff
@@ -1,25 +1,16 @@
 ﻿###Hair colors
```

Comment:
The orcs should have a trapezoid form of the skull. Their head becomes wider to the jaw.
Ignore the left orc on the first one. It's a very old first draft of them.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/121614676-3fc5af00-ca70-11eb-9c10-e05f34f17d7b.png)
![image](https://user-images.githubusercontent.com/46384992/121614760-6edc2080-ca70-11eb-8574-0b60fa69559c.png)

</details>
Right now their skull is like a line.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/121614826-9df29200-ca70-11eb-8739-904ffc7bcd08.png)

</details>

Another part of this problem is that we can't get a Garrosh-like jaw.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/121615036-03468300-ca71-11eb-8864-b0605627de76.png)

</details>

Reply:
I guess I referenced WoW and orcs, not the cinematic orcs. The orcs in WoW don't really have a dramatically wide jaw compared to the width of their head. I could make the slider range have wider jaws, especially for players that want to create their own orcs, but I think it would be an uncommon inheritance rate for randomly spawned pawns. Does that sound reasonable?

Reply:
> I guess I referenced WoW orcs, not the cinematic orcs.

Do you mean player's model? It's barely customizable it shouldn't be your only source.

Reply:
> I think it would be an uncommon inheritance rate for randomly spawned pawns. Does that sound reasonable?

The most of notable orcs have a wide jaw. While Garrosh is the max orcish jaw possible.
<details>
<summary>Click to expand</summary>

Saurfang
![image](https://user-images.githubusercontent.com/46384992/121618591-13ae2c00-ca78-11eb-8b2a-d7c530d1f0aa.png)
Blackhand
![image](https://user-images.githubusercontent.com/46384992/121618637-2de80a00-ca78-11eb-9653-452f9c53f0b5.png)

</details>